presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -125,10 +125,17 @@ class PlanNode extends React.Component<PlanNodeProps, PlanNodeState> {
     }
 
     render() {
+        const details = this.props.details;
+        if (details.includes("Distribution")) {
+            var index = details.indexOf("Distribution");
+            var distribution = details.slice(index, details.length).split("\n")[0];
```

Comment:
I think that including the unprocessed content of the `details` field in the pop-up is more in line with the current UI (keeping 'business logic' out of the view).  But as you pointed out, that also has drawbacks.

Reply:
I agree, it fits the style better. I added a screenshot in another comment that I think is a good compromise.